README.rst

```diff
@@ -134,6 +134,37 @@ Database logger configuration(optional)
    If no log level is set the default is ``debug``.
 
 
+**API Usage**
+
+You can access CKAN harvest logs via the API:
+
+    $ curl {ckan_url}/api/3/action/harvest_log_list
+
+Replace {ckan_url} with the url from your CKAN instance.
+
+Allowed parameters are: 
+
+    * level (filter log records by level)
+
+    * per_page (used for pagination)
```

Comment:
@amercader I updated the pagination limit param name and updated the readme.

You can checkout the new PR [here](https://github.com/ckan/ckanext-harvest/pull/247).